cli/src/template_builder.rs

```diff
@@ -329,6 +329,59 @@ fn build_string_method<'a, L: TemplateLanguage<'a>>(
                 |(haystack, needle)| haystack.contains(&needle),
             ))
         }
+        "starts_with" | "ends_with" => {
+            let starts_with = function.name == "starts_with";
+            let [needle_node] = template_parser::expect_exact_arguments(function)?;
+            let needle_property = expect_plain_text_expression(language, build_ctx, needle_node)?;
+            language.wrap_boolean(TemplateFunction::new(
+                (self_property, needle_property),
+                move |(haystack, needle)| {
+                    if starts_with {
+                        haystack.starts_with(&needle)
+                    } else {
+                        haystack.ends_with(&needle)
+                    }
+                },
```

Comment:
nit: I slightly prefer separate match arm per function name. It will help if we want to build a HashMap of functions to suggest similar functions on typo.